src/views/virtualmachines/list/VirtualMachinesList.tsx

```diff
@@ -145,7 +148,9 @@ const VirtualMachinesList: FC<VirtualMachinesListProps> = ({ kind, namespace })
     !loadingFeatureProxy &&
     loadedColumns;
 
-  if (loaded && isEmpty(unfilteredData)) {
+  const vmsFilteredWithProxy = isProxyPodAlive && selectedFilters.length > 0;
+
+  if (loaded && isEmpty(unfilteredData) && !vmsFilteredWithProxy) {
```

Comment:
Suggestion: Create a variable with a descriptive name for these conditions to make it more readable.

Reply:
added a const with a readable name